include/dts-functions.sh

```diff
@@ -294,19 +290,14 @@ board_config() {
           DASHARO_REL_VER="1.7.2"
           BIOS_LINK_COMM="$FW_STORE_URL/$DASHARO_REL_NAME/v$DASHARO_REL_VER/${DASHARO_REL_NAME}_v${DASHARO_REL_VER}.rom"
           EC_LINK_COMM="$FW_STORE_URL/$DASHARO_REL_NAME/v$DASHARO_REL_VER/${DASHARO_REL_NAME}_ec_v${DASHARO_REL_VER}.rom"
+          CAN_INSTALL_BIOS="true"
```

Comment:
Interesting. Why this model did not have `CAN_INSTALL_BIOS="false"` then? If it does not support auto-installation of bios - it should have been marked before my changes came.

Reply:
I must've missed it when I was implementing MTL

Reply:
I remember now, I didn't want to change how dts-scripts work.
Before implementing MTL I don't think there was any check before that so I only changed how it worked for MTL as it was newly implemented.

Reply:
> so I only changed how it worked for MTL as it was newly implemented.

Did not get it. Did you implement this in some other way?

Reply:
Before implementing config for MTL there wasn't any check.
I only added it because DTS can't install BIOS on MTL but can install EC.
I set it to true by default to not change default behaviour of already existing config.

Reply:
Addressed [here](https://github.com/Dasharo/dts-scripts/compare/8e94cbc7f99ffe845b06f6f742c9b9863dec795e..992062280bbd17d42f45e14d98c3e682971d4ee4)